tests/SpecTests/FunctionalTestCase.php

```diff
@@ -198,6 +198,10 @@ protected function dropTestAndOutcomeCollections()
     {
         $context = $this->getContext();
 
+        if ($context->databaseName === 'admin') {
```

Comment:
This fixes failures when running the crud/db-aggregate tests on the latest MongoDB version.

Reply:
Is latest 4.4.x, or 4.5-dev?

Reply:
Latest is 4.5-dev